src/ProgressOnderwijsUtils/ProgressOnderwijsUtils.csproj

```diff
@@ -22,8 +22,8 @@
     <PackageReference Include="morelinq" version="3.1.1" />
     <PackageReference Include="Newtonsoft.Json" Version="12.0.2" />
     <PackageReference Include="System.ComponentModel.Annotations" Version="4.5.0" />
-    <PackageReference Include="System.Data.SqlClient" Version="4.6.0" />
-    <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="4.5.2" />
+    <PackageReference Include="System.Data.SqlClient" Version="4.7.0-preview5.19224.8" />
+    <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="4.6.0-preview5.19224.8" />

```

Comment:
Waarom?

Reply:
Oh sorry, wat een onzin.

Ik dacht ik probeer dit ook in de upcoming versions, en heb het per ongeluk gecommit.

(Die zijn trouwens zo'n 30% sneller, yay!)

Ik zal het reverten.

Reply:
Ik heb ook alvast Microsoft.Data.SqlClient uitgeprobeerd, en ook die werkte direct probleemloos, net als de previews.

Reply:
(Voor context, 30% sneller is nog steeds best gestoord traag met bergen allocations, maar hey, stap in de goeie richting.)